scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -2,8 +2,12 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("-j", "--nThreads", type=int, help="number of threads", default=None)
+parser.add_argument("-e", "--era", type=str, choices=["2016PreVFP","2016PostVFP"], help="Data set to process", default="2016PostVFP")
```

Comment:
Probably should have a full 2016 option?

Reply:
Yeah, better to wait, adding that would imply other changes which for now would just make the code messier.